suricata.yaml.in

```diff
@@ -1115,6 +1115,11 @@ pcre:
   match-limit: 3500
   match-limit-recursion: 1500
 
+# MQTT topic detection depth
+mqtt:
+  subscribe-topic-match-limit: 100
```

Comment:
I think the default value should be present in the code as well

Reply:
> Hmm, do you mean also use 100 if no value is set, and to have the only way to have unrestricted matching on all topics to manually set this to 0 in the config?

Exactly

Reply:
Sure, makes sense. Can do that in v2 👍🏻